content/en/real_user_monitoring/error_tracking/mobile/android.md

```diff
@@ -39,6 +39,31 @@ If you have not set up the Android SDK yet, follow the [in-app setup instruction
 
 For any given error, you can access the file path, line number, and a code snippet for each frame of the related stack trace.
 
+### Report ANRs
+
+An "Application Not Responding" ([ANR][6]) is an Android-specific type of error that gets triggered when the application is unresponsive for too long.
+
+ANRs are only reported through the RUM SDK (not through Logs).
+
+#### Non-fatal ANR reporting
+Non-fatal ANRs may or may not have led to the application being terminated (crashing).
+
+{{< img src="real_user_monitoring/error_tracking/rum-anr-non-fatal.png" alt="A non-fatal crash report in Error Tracking." >}}
+
+- In the **Error Tracking** page, non-fatal ANRs are grouped under a **single** issue due to their level of noise
+- By default, non-fatal ANRs on *Android 30+* are **disabled** because it would create too much noise over fatal ANRs. However, they non-fatal ANRs are **enabled** on *Android 29 and below* by default because reporting on fatal ANRs is not possible on those versions.
+
+For any Android version, you can override the default setting for reporting non-fatal ANRs by setting `trackNonFatalAnrs` to `true` or `false` when initializing the RUM SDK.
+
+#### Fatal ANRs reporting
+Fatal ANRs are ANRs that result in crashes. They are reported when the application is being unresponsive and leads to the Android OS displaying a popup dialog to the user, who chooses to force quit the app through the popup.
+
+{{< img src="real_user_monitoring/error_tracking/rum-anr-fatal.png" alt="A fatal crash report in Error Tracking." >}}
+
+- In the **Error Tracking** page, fatal ANRs are listed as **individual issues**
```

Comment:
Maybe we could elaborate on this slightly, saying something like:

- In the **Error Tracking** page, fatal ANRs are grouped based on their similarity, which can result into several **individual issues** being created